spyderlib/cli_options.py

```diff
@@ -39,5 +40,8 @@ def get_options():
     parser.add_option('--profile', action='store_true', default=False,
                       help="Profile mode (internal test, "
                            "not related with Python profiling)")
+    parser.add_option('--travis', dest="travis", action='store_true',
+                      default=False,
+                      help="Run a Qtimer to stop the GUI app in travis-ci")
```

Comment:
Mmmh, I think this option's name and help should be rephrased, given that it's going to show when users do `spyder --help`.

What about

    '--test', ....
    help="Closes the application after 30 secs. Useful only for testing purposes".